js/foundation.offcanvas.js

```diff
@@ -280,6 +280,7 @@ class OffCanvas extends Plugin {
    * @param {Object} event - Event object passed from listener.
    * @param {jQuery} trigger - element that triggered the off-canvas to open.
    * @fires OffCanvas#opened
+   * @todo also trigger 'open' event?
```

Comment:
Part of https://github.com/zurb/foundation-sites/pull/10949

Reply:
Right. It's a todo annotation so we don't forget to revisit this (I hope so). =)